### ISSUE TITLE
Remove MonsterInsights Dependency?

### ISSUE THREAD

**Issue by robneu** [jtsternberg/Google-Analytics-Top-Content-Widget#27]
This plugin would be much more interesting/useful if it were standalone, IMO.

**Comment by jtsternberg** [comment]
I agree, and is something I've been wanting to do since I released it, but now appears [it will be necessary](https://github.com/jtsternberg/Google-Analytics-Top-Content-Widget/issues/25) (or will need to unfortunately abandon development of this plugin). If time wasn't an issue, this would have been done years ago.

**Comment by lkraav** [comment]
IMO it should be a pluggable architecture. Have a self-sufficiency mechanism as the default, but provide a way for someone with enough motivation to provide an adapter to whichever upstream analytics config plugin. Having to separately configure x+1 analytics plugins and maintain that is also not a particularly attractive user experience proposition.

**Comment by dxdc** [comment]
@jtsternberg @lkraav Found this in the code:

https://github.com/jtsternberg/Google-Analytics-Top-Content-Widget/blob/644ee50e6f6a62ebc9f3c2d06ac922a98120fa2f/includes/class-ga-top-content.php#L682-L713

Seems there was an earlier version of Yoast, which is called if the Monster dependency is not met. Is this something we could pursue, maybe updating the Yoast version? It seems like a built-in fallback which is already there. (Also related to #25)
